tests/dags/test_dag_parsing.py

```diff
@@ -39,10 +39,26 @@
 }
 
 
+def test_dag_import_errors():
+    # Attempt to load all DAGs in the DAG_FOLDER, and detect import
+    # errors
+    dagbag = DagBag(dag_folder=DAG_FOLDER, include_examples=False)
+
+    dag_errors = []
+    for filename in dagbag.import_errors.keys():
+        dag_errors.append(Path(filename).name)
+    error_string = (",").join(dag_errors)
```

Comment:
PyCharm is telling me we can remove redundant params here 😅 
```suggestion
    error_string = ",".join(dag_errors)
```

Reply:
Ha, I always dislike the look of it without the parens, but that's a personal thing. I'll let PyCharm win this one 😆